app-frontend/src/components/AuthPage/__tests__/LogIn.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import LogIn, { LogInProps } from '../LogIn';
+
+function noop() {}
+
+const props: LogInProps = {
+  loading: false,
+  error: '',
+  authorizationHandler: jest.fn(noop),
+  changeHandler: jest.fn(noop),
+};
+
+const setUp = () => shallow(
+  <LogIn
+    loading={props.loading}
+    error={props.error}
+    authorizationHandler={props.authorizationHandler}
+    changeHandler={props.changeHandler}
+  />,
+);
+
+describe('<LogIn />', () => {
+  it('Should call an authorization function after pressing Log in', () => {
+    const component = setUp();
+    const logInButton = component.find('.authorize-button');
```

Comment:
Just to know,
You can search your components int he following way: 
    const logInButton = component.find(LogIn);

Reply:
Okay, I will take on board